src/ConstantOptimization.jl

```diff
@@ -1,19 +1,24 @@
 using Optim
 
 # Proxy function for optimization
-function optFunc(x::Array{Float32, 1}, X::Array{Float32, 2}, y::Array{Float32, 1}, baseline::Float32, tree::Node, options::Options)::Float32
+function optFunc(x::Vector{ConstantType}, X::AbstractMatrix{T},
```

Comment:
I think the `Abstract` has been left off here...

Reply:
I think these are okay because these are fixed internal types. I use this vector to get/set the constants in an equation when doing optimization. Thoughts?

Reply:
If it's all internal then I agree it's probably unimportant.